python/CMakeLists.txt

```diff
@@ -22,13 +22,20 @@ endif()
 set(f3d_python_package_name "f3d")
 set(f3d_python_package_suffix "python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages")
 set(f3d_python_lib_output "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${f3d_python_package_suffix}/${f3d_python_package_name}")
+set(f3d_module_dir "${CMAKE_BINARY_DIR}$<${F3D_MULTI_CONFIG_GENERATOR}:/$<CONFIG>>/f3d")
 
 set_target_properties(pyf3d PROPERTIES
   CXX_STANDARD 14
   CXX_VISIBILITY_PRESET hidden
-  LIBRARY_OUTPUT_DIRECTORY "${f3d_python_lib_output}"
+  LIBRARY_OUTPUT_DIRECTORY "${f3d_module_dir}"
   )
 
+if(APPLE)
+  set_target_properties(pyf3d PROPERTIES INSTALL_RPATH "@loader_path/${CMAKE_INSTALL_LIBDIR}")
+elseif(UNIX)
+  set_target_properties(pyf3d PROPERTIES INSTALL_RPATH "$ORIGIN/${CMAKE_INSTALL_LIBDIR}")
```

Comment:
in that case isnt ORIGIN all the way to `lib/python3.11/site-packages/f3d` ?

Reply:
why would dependencies be in site-packages ? there should be in install/lib directly . Unless you are talking about a specific way to build with wheels ?

Reply:
If you build wheels (`SKBUILD` defined), then everything is installed in `<prefix>/lib/python3.11/site-packages/f3d`.  
If not, you are right, dependencies are located in `<prefix>/lib` but in this case, I suppose `<prefix>/lib` does not have to be specified because I'd expect it to be a standard path (e.g. `/usr/lib`)

Reply:
would it be possible to add this only when building wheels ?

Reply:
actually, I messed up. The same code is below, only if `SKBUILD` is enabled, so let me just delete this block.